library/ssl_msg.c

```diff
@@ -3525,6 +3525,14 @@ static int ssl_check_record_type( uint8_t record_type )
     return( 0 );
 }
 
+static int ssl_parse_record_header( mbedtls_ssl_context const *ssl,
+                                    unsigned char *buf,
+                                    size_t len,
+                                    mbedtls_record *rec )
+{
+    /* Check TLS version */
+}
+
```

Comment:
Why is this code snippet here (with an empty function implementation) when the same function is implemented a few lines below?

Reply:
That's a mistake.